hugging_face_classifier_trainer/hugging_face_classifier_trainer.py

```diff
@@ -692,12 +711,15 @@ def preprocess_function(examples):
             label_columns=label_name,
             drop_columns=drop_columns,
         )
-
         train_dataset, test_dataset = train_test_split(
             dataset, test_size=train_test_split_size, random_state=random_state
         )
         train_dataset = Dataset.from_pandas(train_dataset)
         test_dataset = Dataset.from_pandas(test_dataset)
+    else:
+        raise mlrun.errors.MLRunInvalidArgumentError(
+            "You have to provide 'hf_dataset' or 'dataset'"
```

Comment:
A good exception should print the current errored state -> tell why it was raised -> then, if possible, tell how to overcome it.

So, in this case for example:
```suggestion
            "Training data was not provided. A training dataset is mandatory for training. Please provide a training set using one of the arguments 'hf_dataset' or 'dataset'."
```